flasksrc/session.py

```diff
@@ -77,6 +78,15 @@ class ToyNetSessionByIdCreateSwitchPutResp(Schema):
     pass
 
 
+class ToyNetSessionByIdCreateLinkPutReq(Schema):
+    dev_1 = fields.Str()
+    dev_2 = fields.Str()
+
+
+class ToyNetSessionByIdCreateLinkPutResp(Schema):
+    pass
```

Comment:
What is the purpose for not implementing this? Is there an expected future use?

Reply:
Partially to enable easy modification should the API change, partially to ensure that Swagger generates complete documentation (even containing empty entries), and partially to guard against errors of omission or perceived errors of omission.